cocos/core/math/color.ts

```diff
@@ -549,20 +555,21 @@ export class Color extends ValueType implements Modifiable {
      * ```
      */
     public toHEX (fmt: '#rgb' | '#rrggbb' | '#rrggbbaa' = '#rrggbb'): string {
+        const data = this._data;
```

Comment:
It's better to use `thisData` to keep compatible.

Reply:
Okay.